spacy/pipeline/entity_linker.py

```diff
@@ -41,6 +41,14 @@
 DEFAULT_NEL_MODEL = Config().from_str(default_model_config)["model"]
 
 
+"""
+    [initialize.components.entity_linker.kb_loader]
+    @misc = "spacy.CustomKB.v1"
+    entity_vector_length = 342
+    custom_field = 666
+"""
```

Comment:
```suggestion
"""
[initialize.components.entity_linker.kb_loader]
@misc = "spacy.CustomKB.v1"
entity_vector_length = 342
custom_field = 666
"""
```

Is there a reason this is indented?

Reply:
No, just a typo.